.github/workflows/deploy-docs.yaml

```diff
@@ -0,0 +1,73 @@
+name: Deploy docs
+
+on:
+  release:
+    branches:
+      - master
+    types:
+      - published
+  pull_request:
+    branches:
+      - master
+      - development
```

Comment:
Do we really need to build the docs on PRs?
Or is it just for testing if it builds the docs?

Reply:
IMO this is a good way to ensure valid commits for all PRs. this will only build them. Publishing happens only on Tags on master

Reply:
Fine for me. Question was just for clearification.